src/pages/blog/[slug].js

```
@@ -27,54 +27,119 @@ const slugToPostContent = ((postContents) => {
+        />
+
+        {scripts &&
+          (typeof scripts === "string"
+            ? extractScripts(scripts).map((script, idx) => (
+                <script
+                  key={idx}
+                  // Optionally parse type attribute, etc. from script.attrs if needed
+                  type={
+                    /type=["']([^"']+)["']/.exec(script.attrs)?.[1] || undefined
+                  }
+                  dangerouslySetInnerHTML={{ __html: script.content }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)